backend/src/libs/plugins/authorization/authorization.plugin.ts

```diff
@@ -38,16 +38,21 @@ const authorization = fp<Options>((fastify, { services }, done) => {
     }
 
     const { jwtService, userService } = services;
-    const { userId } = await jwtService.decode<AuthTokenPayload>(token);
-    const authorizedUser = await userService.findById(userId);
-
-    if (!authorizedUser) {
+    try {
+      const { userId } = await jwtService.decode<AuthTokenPayload>(token);
+      const authorizedUser = await userService.findById(userId);
+      if (!authorizedUser) {
+        throw new AuthError({
+          message: ExceptionMessage.INVALID_TOKEN,
+        });
+      }
+
+      request.user = authorizedUser;
+    } catch {
       throw new AuthError({
         message: ExceptionMessage.INVALID_TOKEN,
```

Comment:
Let's change message to ExceptionMessage.UNAUTHORIZED_USER

Reply:
done